app/src/main/assets/licenses.xml

```diff
@@ -108,4 +108,40 @@
         website="https://github.com/shts/TriangleLabelView"
         license="Apache 2.0"
         licenseText="LICENSE_TRIANGLE_LABEL_VIEW.txt" />
+    <library
+        name="Conscrypt"
```

Comment:
The licenses are currently sorted alphabetically. Please keep them sorted.

Reply:
Didn't notice that, will do.